src/network/serverpackethandler.cpp

```diff
@@ -109,11 +110,11 @@ void Server::handleCommand_Init(NetworkPacket* pkt)
 		depl_serial_v = SER_FMT_VER_INVALID;
 
 	if (depl_serial_v == SER_FMT_VER_INVALID) {
-		actionstream << "Server: A mismatched client tried to connect from "
-				<< addr_s << std::endl;
-		infostream<<"Server: Cannot negotiate serialization version with "
-				<< addr_s << std::endl;
-		DenyAccess(pkt->getPeerId(), SERVER_ACCESSDENIED_WRONG_VERSION);
+		actionstream << "Server: A mismatched client tried to connect from " <<
+			addr_s << std::endl;
+		infostream << "Server: Cannot negotiate serialization version with " <<
+			addr_s << std::endl;
```

Comment:
should be made more useful or removed, my suggestion:
```suggestion
			addr_s << " client_max=" << (int)client_max << std::endl;
```